docs/source-app/examples/github_repo_runner/app.py

```diff
@@ -56,7 +56,8 @@ def run(self, *args, **kwargs):
         # 2: Use git command line to clone the repo.
         repo_name = self.github_repo.split("/")[-1].replace(".git", "")
         cwd = os.path.dirname(__file__)
-        subprocess.Popen(f"git clone {self.github_repo}", cwd=cwd, shell=True).wait()
+        subprocess.Popen(
+            f"git clone {self.github_repo}", cwd=cwd, shell=True).wait()
```

Comment:
should install a code-formatter and also in the CI.

Reply:
@tchaton ?

Reply:
@rohitgr7 The code is optimized for mobile rendering. We need to have a custom black formatting at 80c per line for the App docs.

Reply:
Done !